src/resmom/mom_hook_func.c

```diff
@@ -638,6 +638,39 @@ vnl_add_vnode_entries(vnl_t *vnl, vmpiprocs *vnode_entry, int num_vnodes,
 	return (0);
 }
 
+/**
+ * @brief
+ *	Duplicates pointer to hooks parameters.
+ *
+ * @param[in]   php - structure for duplication
+ *
+ * @return new_php	for success
+ * @return NULL		for error
+ *
+ */
+mom_process_hooks_params_t
+*duplicate_php(mom_process_hooks_params_t *php) {
```

Comment:
please move the "{" to the first column of the next line.

Reply:
Done